crates/astria-core/src/protocol/bridge/v1alpha1/mod.rs

```diff
@@ -22,25 +24,27 @@ impl BridgeAccountLastTxHashResponse {
     ///
     /// - if the transaction hash is not 32 bytes
     pub fn try_from_raw(
-        raw: raw::BridgeAccountLastTxHashResponse,
+        raw: &raw::BridgeAccountLastTxHashResponse,
```

Comment:
Please keep the argument taking ownership.

If you want an argument taking a reference, add `try_from_raw_ref`